troubleshooting/faq.md

```diff
@@ -161,6 +162,26 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## What should I do with git dependencies on closed networks?
+
+If you are installing the package that have a dependency on a git url, npm will fetch that package from github itself because git dependencies are valid in npm.
```

Comment:
a couple edits:

rather than:

`If you are installing the package that have a dependency on a git url`

let's go with:


`If you are installing a package that has a git URL as a dependency, npm will fetch that package from the remote git repository itself.`

rather than

`npm will fetch that package from github itself because git dependencies are valid in npm.`

Let's go with:

`. npm supports both git and HTTP URL formats (see https://docs.npmjs.com/files/package.json#dependencies), so we will need to work around this in our private registry.`